iamcristos/politico-app
# Ft get election result endpoint 163749505

#### What does this PR do?
GET endpoint

#### Description of Task to be completed?
Have this endpoint working so as to edit all registerd political offices
/GET '/api/v1/:id/result'
#### How should this be manually tested?
clone this reprository run npm open this url on postman GET request and all candidate result for a specific office should appear
 
#### Any background context you want to provide?
if there are no election returns error
#### What are the relevant pivotal tracker stories?
#163749505
#### Screenshots (if appropriate)
#### Questions:
